src/App.js

```diff
@@ -1,12 +1,12 @@
-import Calculator from './components/Calculator';
+import CalculatorHooks from './components/CalculatorHooks';
```

Comment:
[OPTIONAL] I would recommend you remove the original `Calculator` component from the repository. If you ever need some piece of code from this component you can find it in the `GitHub` historical.